rm/rm-server/src/main/java/org/ow2/proactive/utils/appenders/AsynchChachedFileAppender.java

```diff
@@ -42,26 +42,30 @@
     @Override
     public void append(String cacheKey, LoggingEvent event) {
```

Comment:
Maybe add `@SuppressWarnings("squid:S2142")` since we purposely chose to not reiterate the Interruption state and/or exception